common/src/main/java/org/wso2/testgrid/common/DashboardSetup.java

```diff
@@ -71,6 +76,7 @@ public void initDashboard() {
             String dbName = testplanID;
             influxDB.createDatabase(dbName);
             influxDB.close();
+            logger.info("database created for performance data");
```

Comment:
This log is insufficient. A log must contain the context like what db you created, for what test plan id etc.